serial_driver/src/serial_bridge_node.cpp

```diff
@@ -126,21 +126,36 @@ void SerialBridgeNode::get_params()
   auto sb = StopBits::ONE;
 
   try {
+// NOTE(esteve): remove this when Foxy is no longer supported
+#if RCLCPP_VERSION_MAJOR > 2
     m_device_name = declare_parameter<std::string>("device_name");
+#else
+    m_device_name = declare_parameter<std::string>("device_name", "");
+#endif
```

Comment:
Couldn't you just use the defaulted signature for all ROS versions and avoid the macros?

Reply:
That's fine too, I just wanted to keep the code as close as possible to the current state for Galactic.

Reply:
Done in 5c43956

Reply:
Thanks!